locales/en/email-strings.ftl

```diff
@@ -3,6 +3,11 @@
 # Firefox is a brand name and should not be translated.
 -brand-name = Firefox
 
+# Firefox Relay is a product name and should not be translated.
+-product-name-relay = Firefox Relay
+# Mozilla VPN is a product name and likely should not be translated.
```

Comment:
Let's drop the "likely". We have rules for brand names, so what's doable should be already clear.